docs/advanced/general-security.md

```diff
@@ -0,0 +1,37 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Harddrive.svg" /> Security
+
+This page provides a list of things you can do to improve your security when working with DCR, or cryptocurrencies more generally.
+
+
+## System setup
+
+Here are some measures you can take when setting up your hardware and software to make your system more secure. 
+
+- **Disk encryption:** pretty much every OS supports disk encryption of some kind. Use it to secure any machines you use, so that they are protected against trivial theft. Note that a sophisticated adversary can extract your disk encryption keys from your device's RAM if they can access your machine while powered on.
```

Comment:
Instead of 

> so that they are protected against trivial theft

would prefer something like 

> to help protect against trivial theft

Disk encryption is not ultimate protection, but it does help.

Reply:
Reworded to:

> - **Disk encryption:** Nearly all Operating Systems (OSes) support disk encryption of some kind. While not an ultimate protection, it does help. Note that a sophisticated adversary can extract your disk encryption keys from your device's RAM if they can access your machine while powered on.